Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionResultPositiveOtherWarningFragment.kt

```diff
@@ -126,6 +107,12 @@ class SubmissionResultPositiveOtherWarningFragment : Fragment(),
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        internalExposureNotificationPermissionHelper =
+            InternalExposureNotificationPermissionHelper(this, this)
+        requireActivity().onBackPressedDispatcher.addCallback(viewLifecycleOwner, backCallback)
+        binding.submissionViewModel = submissionViewModel
```

Comment:
How about keeping the order consistent. Binding init before other calls.

Reply:
I just copied pasted that, but totally agree here, when already touching this, should make it nice.